doc/src/input/systems/interactions/potentials.rst

```diff
@@ -208,3 +208,30 @@ and ``b`` (well width) should be provided as strings.
 
 .. caution::
     ``b`` has to be positive
+
+
+Mie potential
+-------------
+
+The Mie potential is a classical potential for pair interactions
+expressed as:
+
+.. math::
+
+    V(r) = \frac{n}{n-m} \left(\frac{n}{m}\right)^{m/(n-m)}\epsilon
+    \left[ \left( \frac{\sigma}{r}\right)^n - \left( \frac{\sigma}{r}\right)^m \right]
+
+The potential type keyword is ``mie`` and the parameters are
+``sigma`` (:math:`\sigma`), the particle diameter, and ``epsilon`` (:math:`\epsilon`), the energetic
+parameter, which should be provided as strings (with units).
+The repulsive exponent ``n`` and the attractive exponent ``m`` should be provided as
+floats.
```

Comment:
We could accept numbers for `m` and `n`, and then convert to a float in the input if an integer is passed.